ABWL-LEDER/appendix/LKF.md

```diff
@@ -32,7 +32,8 @@ Organisation der Durchführung von Prozessen
 
 #### Sie können Aufgabenanalyse und –Synthese zum Erstellen der Inhalte einer Aufbauorganisation erläutern
 
-- TODO
+- **Aufgabenanalyse:** Ermitteln von **elementaren (Teil-)Aufgaben** $\rightarrow$ fachlich und zeitlich **optimale Zusammenfassung/Bündelung zu Stelle**
+- **Aufgabensynthese:** ökonomisch sinnvolle Aufgaben bündeln (von Stellen) zu Abteilungen bis hin zur Gestaltung der gesamten Aufbauorganisation $\rightarrow$ rationell gestaltete Arbeitsteilung im Unternehmen
 
 #### Sie können Aufgabenanalyse und –Synthese zum Erstellen der Inhalte einer Ablauforganisation erläutern
 
```

Comment:
```suggestion
```

Die Frage ist gerade doppelt drin

Reply:
Ich sag, wir nehmen die Frage einfach raus.

Reply:
Ich würde vorschlagen die Frage drinnen zu lassen, aber es in Aufbauorg abzuändern

Reply:
@sthinbetween Die identische Frage zur Aufbauorganisation steht schon drüber 👍